### ISSUE TITLE
Account settings - message archiving toggles unresponsive

### ISSUE THREAD

**Issue by licaon-kter** [tigase/stork#11]
...(Enable, AutoSync, Sync) unless I scroll the view after the first touch

3.0.23.p
Xiaomi MIUI11 Android 10

**Comment by woj-tek** [comment]
Is it marks as disabled (greyed out) or tapping on it doesn't work? Does your server generate response to enabling?

**Comment by licaon-kter** [comment]
Nope, it is grey out when disconnected, right? While this happens a lot (https://github.com/tigase/stork/issues/3) it's not the issue here.

> Does your server generate response to enabling?

I'm on the log level under Debug, and ejabberd does not generate this info, I'd rather not go in stanza-land with level Debug. :)

Video:
[31761.zip](https://github.com/tigase/stork/files/3926671/31761.zip)

**Comment by woj-tek** [comment]
OK, from what I can see the UI blocks until response from the first change is received, which IMHO makes sense, and it seems there is a rather longer delay receiving response from the server hence perceived "unresponsiveness"?

Maybe adding progressbar (something along the lines of https://material.io/components/progress-indicators/) would be more informative that's something is happening?

**Comment by licaon-kter** [comment]
Do note that if I scroll the view just a little bit right away it will instantly toggle, not sure it's dependant on server.

**Comment by woj-tek** [comment]
I tried to reproduce the issue and indeed there is something wrong with the view - it works the first time (I.e. open account settings, toggle it, go back, open settings again, it switches immediately). But it "get stuck" if you try to toggle it more than once without leaving the view.
